backend/tournesol/tests/test_exports.py

```diff
@@ -193,32 +195,35 @@ def test_not_authenticated_can_download_all_exports(self):
         self.assertEqual(response.headers["Content-Type"], "application/zip")
 
         content_disposition = response.headers["Content-Disposition"]
-        match = re.search("attachment; filename=(tournesol_export_\\d{8}T\\d{6}Z).zip", content_disposition)
+        match = re.search(
+            "attachment; filename=(tournesol_export_\\d{8}T\\d{6}Z).zip",
+            content_disposition,
+        )
         self.assertIsNotNone(match)
         root = match.group(1)
 
         zip_content = io.BytesIO(response.content)
-        with zipfile.ZipFile(zip_content, 'r') as zip_file:
+        with zipfile.ZipFile(zip_content, "r") as zip_file:
             expected_files = [
                 root + "/README.txt",
                 root + "/comparisons.csv",
                 root + "/users.csv",
             ]
             self.assertEqual(zip_file.namelist(), expected_files)
 
-            with zip_file.open(root + '/README.txt', 'r') as file:
+            with zip_file.open(root + "/README.txt", "r") as file:
                 content = file.read()
-                with open('tournesol/resources/export_readme.txt', 'rb') as readme_file:
+                with open("tournesol/resources/export_readme.txt", "rb") as readme_file:
                     expected_content = readme_file.read()
                 self.assertEqual(content, expected_content)
 
-            with zip_file.open(root + '/comparisons.csv', 'r') as file:
+            with zip_file.open(root + "/comparisons.csv", "r") as file:
                 content = file.read()
                 expected_content = self.client.get("/exports/comparisons/").content
                 self.assertEqual(content, expected_content)
 
-            with zip_file.open(root + '/users.csv', 'r') as file:
-                content = file.read().decode('utf-8')
+            with zip_file.open(root + "/users.csv", "r") as file:
+                content = file.read().decode("utf-8")
```

Comment:
Want to enforce something (recommended editor settings / pre-commit hooks / black --check in CI / ...) to avoid this kind of diffs? It makes review slightly harder and has a small chance that a reviewer miss a relevant change hidden among syntax changes.

Reply:
Relevant bit from our readme about the pre-commit hooks:
```
## F.A.Q.

**Why are the code quality tools not automatically triggered locally with the
help of the Git pre-commit hook?**

We try to use the [pre-commit][pre-commit] framework, but unfortunately it's
not adapted to work smoothly within our mono-repository
configuration [[faq-1][faq-1]].

Each time we found a solution a new issue appeared. In the end it requires much
more efforts to be configured and used rather than simply running the code
quality checks manually.

As we do not want to make the developers to have complex and error-prone local
setups, we decided to add the code quality checks only in the CI for now.
```